packages/server/lib/util/settings.js

```diff
@@ -3,7 +3,31 @@ const Promise = require('bluebird')
 const path = require('path')
 const errors = require('../errors')
 const log = require('../log')
-const { fs } = require('../util/fs')
+const { fs } = require('./fs')
+const requireAsync = require('./require_async').default
+const debug = require('debug')('cypress:server:settings')
+
+function jsCode (obj) {
+  const objJSON = obj && obj !== {} ? JSON.stringify(obj, null, 2) : `{
```

Comment:
In e2e tests we (ab)use the _write function to setup a config file

Reply:
But testing `obj && obj !== {}` is the same as saying `obj && true` because `obj` can never `===` an object defined inline `{}`, right? It seems like the `obj !== {}` bit is either unnecessary or incorrect.

Reply:
We check for an empty object here so when we get an empty object, is formatting of the output code. 
We return the code with a return in the middle instead of inline `{}`. Do you think my comparison with empty object will never match?

[EDIT] Objects are matched by reference in JS (duh!) my mistake. I need to fix this. good catch

Reply:
Done in 0dd3649ab1d8a773ac460eef2d8b2618b01f4ea0